### DISCUSSION TITLE
Compatibility with Series Feature in The Event Calendar Pro

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Revisions#922] [Feature Requests]
>  In The Events Calendar Pro 6.0 they introduced the ability to create a series of events. As an admin, I am able to click on an "Edit All" link when looking at the events in the Events backend list, and that will allow me to edit the entire series by altering the basis tribe_event of the series. However, as a revisor, who wants to make a revision to the entire series, I do not see a "create revision" type link for the entire series, unless an admin turns on "Edit Published Events" in the Capabilities plugin, which lets revisors approve their revisions, which is not desirable. I think it's because the "edit all" is custom TEC functionality, and Revisions cannot "see" it. Is there a plan to add a feature so that it would be possible to submit a revision for an entire series of events? Thanks!
> 
> Here's some screenshots of the front end "Edit all" and the HTML for the "Edit all" link.
> 
> Then for the actual plugin code, I don't think I should share that code because it belongs to The Events Calendar, but I think if you have the latest version of the plugin you could look around line 352 of wp-content/plugins/events-calendar-pro/src/Tribe/Recurrence/Meta.php to see how the code is checking if the current author can edit the first event in the series and then building a link to "edit series url" which just uses the same first event in series.
> 
> There is also a section below that ending around line 382 that deals with how trash and delete actions can act on the whole series of events.
> 
> That entire Reccurence folder could be useful to look at.

![image](https://user-images.githubusercontent.com/83811723/216254850-a1ac2c7b-b02b-4e8a-9934-e97bd74189f4.png)

![image](https://user-images.githubusercontent.com/83811723/216254906-31d75d73-ca92-48ca-a786-bccb3e3ba0ef.png)

**Comment by niceindividual** [comment]
If any developers have questions about this, or need a tester, I am here for it! Following this thread.
